xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -299,9 +303,11 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the maximum amount of data that can be drained from responses in bytes.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets the maximum amount of data that can be drained from responses in bytes. Draining occurs when a request is cancelled or response is disposed prior to fully reading content. If the number of bytes drained exceeds this amount, the connection will be closed rather than reused.</summary>
```

Reply:
The two sentences starting with "Draining occurs..." should appear in the remarks, not the summary.